.github/workflows/ci.yml

```diff
@@ -707,6 +709,30 @@ jobs:
             -Dtest.redshift.aws.region="${AWS_REGION}" \
             -Dtest.redshift.aws.access-key="${AWS_ACCESS_KEY_ID}" \
             -Dtest.redshift.aws.secret-key="${AWS_SECRET_ACCESS_KEY}"
+      - name: Fault Tolerant Redshift Tests
```

Comment:
optional: The non FTE tests for Redshift are called "Cloud Redsfhift tests" - maybe call these "Cloud Fault Tolerant Redshift Tests"

Reply:
We want to use different redshift clusters for each set of tests. 

The redshift tests are prone to timing out as @findinpath mentioned and adding more tests against the same cluster will make that more likely to happen.

Reply:
each single CI job uses a single cluster. So having multiple tests run in single profile only increases the wall time of the tests. It doesn't affect flakiness at all.

FTE tests are peculiar though because they use a semaphore to bypass the global concurrency limits. If they cause issues the concurrency can be limited (see the ctor of BaseFailureRecoveryTest iirc). Multiple clusters per PR adds non-negligible cost increase.

Reply:
> adding more tests against the same cluster will make that more likely to happen.

By `Can you just modify Cloud Redshift Tests to pass proper -pl :... based on the test matrix?` I did not mean to share the CI job to run fault tolerant and not fault tolerant tests. 
But to instantiate two versions of "Cloud Redshift Tests"; one for normal tests; one for FTE.

Reply:
Gotcha, yes that is possible. 

Will set the step name dynamically based on the profile to disambiguate between the fte tests/cloud tests.